pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -121,6 +120,17 @@ func minioEnvironmentVars(t *miniov2.Tenant, wsSecret *v1.Secret, hostsTemplate
 			},
 		})
 	}
+	// Check if any domains are configured
+	if t.Spec.Features != nil && t.Spec.Features.Domains != nil && len(t.Spec.Features.Domains.Minio) > 0 {
```

Comment:
we can abstract this to a `HasDomains` function or something similar

Reply:
done